admin/base/utils.py

```diff
@@ -0,0 +1,22 @@
+"""
+Utility functions and classes
+"""
+from django.contrib.auth.views import redirect_to_login
+from django.core.exceptions import PermissionDenied
+from django.contrib.auth.mixins import UserPassesTestMixin
+
+
+class OSFAdmin(UserPassesTestMixin):
+    login_url = '/admin/auth/login'
```

Comment:
Is this necessary? Aside from the lack of a trailing slash, it's the same as `admin_settings.LOGIN_URL`.